CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.12.1] - 2021-07-06
+
+### Fixed
+- prevent thruty value on condition
+
```

Comment:
There's no need to manually input the version number here, the CI/CD bot will take care of it :)

```suggestion
## [Unreleased]
### Fixed
- prevent thruty value on condition

```